File: TP1/SayHello_OC/tme1/ViewController.m
//
//  ViewController.m
//  tme1
//
//  Created by m2sar on 05/10/2020.
//  Copyright © 2020 Sorbonne Université. All rights reserved.
//

#import "ViewController.h"
int b=0;
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    [[self l] setText:@""];
    [[self button] setTitle:@"SayHello" forState:UIControlStateNormal];
}


- (IBAction)action:(id)sender {
    if (b==0){
        [[self l] setText:@"Hello"];
        [[self button] setTitle:@"SayBye" forState:UIControlStateNormal];
        b=1;
    }else{
        [[self l] setText:@"Bye Bye"];
        [[self button] setTitle:@"SayHello" forState:UIControlStateNormal];
        b=0;
    }
}
@end

File: TP1/SayHelloSwift/SayHelloSwift/ViewController.swift
 //
//  ViewController.swift
//  SayHelloSwift
//
//  Created by m2sar on 07/10/2020.
//  Copyright © 2020 Sorbonne Universite. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    
    @IBOutlet weak var l: UILabel!
    @IBOutlet weak var button: UIButton!
    var b = 0
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.l.text = ""
    }

    @IBAction func action(_ sender: UIButton) {
        if(b == 0){
            self.button.setTitle("SayBye", for : .normal)
            self.l.text = "Hello"
            self.b = 1
        }else{
            self.button.setTitle("SayHello", for : .normal)
            self.l.text = "ByeBye"
            self.b = 0
        }
        
    }
    
}

